src/common/console/c_console.cpp

```diff
@@ -250,6 +256,18 @@ void C_InitConsole (int width, int height, bool ingame)
 	CmdLine.ConCols = ConWidth / cwidth;
 
 	if (conbuffer == NULL) conbuffer = new FConsoleBuffer;
+
+	i = Args->CheckParm ("-refreshfile");
+	if (i > 0 && i < (int)Args->NumArgs() - 1)
+	{
+		RefreshFile = Args->GetArgList(i + 1);
+		RefreshInterval = TICRATE;
```

Comment:
When trying to merge this into QZDoom, I get the error that TICRATE isn't found.

Reply:
Confirmed, when doing this just on GZDoom, TICRATE isn't found. Please fix this.

Reply:
Which toolchain are you using? It builds fine on my machine with GCC 11.2.0.
I have just added an `#include "doomdef.h"` statement in c_console.cpp, it should fix the problem, TICRATE is defined as enum in this file. Please try to reproduce the issue by building 315e58f.

Reply:
I am using Visual Studio 2017. I'll try it again in a bit. If it works, then I'll put it in QZDoom.